### PR TITLE
Deprecate push -m

### PR DESCRIPTION
**Pull request by cbednarski**

### CODE REVIEW

**Comment by mitchellh** [comment]
Instead of logging let's use the actual UI mechanism to warn the user.

File: command/push.go [hunk 1]
```diff
@@ -53,6 +54,10 @@ func (c *PushCommand) Run(args []string) int {
 		return 1
 	}
 
+	if message != "" {
+		log.Printf("[WARN] -m/-message is deprecated and will be removed in a future Packer release")
```
Comment (sethvargo) [inline command/push.go]: Do we have a ui object we could use instead?

Comment (mitchellh) [inline command/push.go]: Yeah, its available, I said that as well in my comment :)

Comment (sethvargo) [inline command/push.go]: oh lol. I didn't see your comment

**Pull request merged**